packages/quick_blue/lib/src/quick_blue_linux.dart

```diff
@@ -34,6 +35,12 @@ class QuickBlueLinux extends QuickBluePlatform {
       }
       _client.deviceAdded.listen(_onDeviceAdd);
 
+      _activeAdapter?.propertiesChanged.listen((event) {
+        if (_oldState == _getAvailabilityState) return;
```

Comment:
Is it necessary?

Reply:
@Sunbreak yes because `propertiesChanged ` stream will be triggered when any of the property will change for that Adapter , including Power on/off , [here](https://github.com/canonical/bluez.dart/blob/main/example/monitor.dart#L41) is list of Properties which will affect this stream , 
and whole idea is to check adapter state if its Powered on or not , whenever this stream updates , and to prevent sending duplicate events , added this condition here

Reply:
https://github.com/woodemi/quick.flutter/blob/0ccb6d4ebcf338dffe02b1668e7a2a6c7a4c107f/packages/quick_blue/lib/src/quick_blue_linux.dart#L156-L161

When the value of `BlueZGattCharacteristic` changed, there is a key of `Value` in the fired event

We'd better filter the `propertiesChanged` event of `_activeAdapter` as well

Reply:
@Sunbreak ok got it